pallets/substrate-app/src/lib.rs

```diff
@@ -350,6 +350,7 @@ pub mod pallet {
             asset_id: AssetIdOf<T>,
             sidechain_asset: ParachainAssetId,
             allowed_parachains: Vec<u32>,
+            min_amount: BalanceOf<T>,
```

Comment:
It's not clear from the name that it is a minimal xcm transferable amount. I suggest to rename to `minimal_xcm_amount`.

Reply:
Done